website/content/api-docs/agent/check.mdx

```diff
@@ -237,6 +237,8 @@ The table below shows this endpoint's support for
   made to both addresses, and the first successful connection attempt will
   result in a successful check.
 
+- `UDP` `(string: "")` - Specifies a `UDP` IP address, hostname, port, or combination thereof. The check sends datagrams to the value specified at the interval specified in the `Interval` configuration. If the datagram is sent successfully or a timeout is returned, the check is set to the `passing` state.  The check is logged as `critical' if the datagram is sent unsuccessfully.
```

Comment:
> Specifies a `UDP` IP address, hostname, port, or combination thereof.

@fulviodenza, @dhiaayachi, @trujillo-adam: Is this wording accurate? For example, can this string _just_ be a port? To me, this wording seems like any component can be used separately, and any combination of them is possible.

I think it would be clearer to use the same wording as used below: https://github.com/hashicorp/consul/pull/12722/files#diff-fde54c45fa092a8b19350f419572bdfa28505a986952361ca683d127ef7a9663R88-R89

There's also a small mistake: critical has ` on the left, but ' on the right.

I propose the following:
```suggestion
- `UDP` `(string: "")` - Specifies a `UDP` IP address/hostname and port.
The check sends datagrams to the value specified at the interval specified in the `Interval` configuration.
If the datagram is sent successfully or a timeout is returned, the check is set to the `passing` state.
The check is logged as `critical` if the datagram is sent unsuccessfully.
```

Thoughts?